Library/Formula/symlinks.rb

```diff
@@ -0,0 +1,23 @@
+class Symlinks < Formula
+  desc "scan/change symbolic links"
+  homepage "http://www.ibiblio.org/pub/Linux/utils/file/symlinks.lsm"
```

Comment:
Yeah, that's one of those odd cases where there is no proper homepage (at least none I know of), but the utility itself is quite popular and exists in every Linux distribution I used so far. Linking to the Debian package page is a bit problematic in that it is specific to the Debian release.